doc/distribution.md

```diff
@@ -0,0 +1,104 @@
+# Content Distribution
+
+When an editor publishes content in Writer, it gets distributed across the web. In this doc, we break down the distribution of this content.
+
+![distribute-content-meme](http://files.artsy.net/images/screen-shot-2017-05-25-at-40907-pm.png)
+
+## [Artsy.net](https://www.artsy.net/articles)
+
+We publish all of our content on [Artsy.net](https://www.artsy.net/articles). Each piece of content belongs to a Channel which helps us decide how an article should look.
+
+Because we have multiple channels, each piece of content on the site gets treated differently depending on which channel it belongs to. For example, we have [promoted content](https://www.artsy.net/article/artsy-school-s-out-for-the-summer-paul-winstanley-s-spare-portraits-of-empty-art-schools) written by Artsy staff, [partner-written articles](https://www.artsy.net/waterhouse-and-dodd/article/waterhouse-dodd-interview-juliette-losq), and [Life at Artsy](https://www.artsy.net/article/daniel-doubrovkine-what-do-software-engineers-do) posts which you can see all have different visual treatments.
+
+In [Force](https://github.com/artsy/force), we've separated the main article body rendering to a [component](https://github.com/artsy/force/tree/master/desktop/components/article). This helps us embed articles anywhere on the site, like on a [fair article](https://www.artsy.net/the-armory-show-2017/info/about-the-fair) or artist bio.
+
+>##### Aside on Channels
+> We typically think of Channels as groups of people. A [Channel](https://github.com/artsy/positron/blob/master/api/apps/channels/model.coffee) contains some metadata and an array of Users. Users can be in multiple Channels. Depending on which Channel you're writing articles in, you'll see different layout options and administrative settings. A long time ago, we only had Users and our entire Artsy Editorial staff had to use a single login.
+
+Bored? Here are some great Artsy Editorial (Channel) articles:
+- [Are Video Games Art?](https://www.artsy.net/article/artsy-editorial-are-video-games-art)
+- [What Happens If You Break an Artwork?](https://www.artsy.net/article/artsy-editorial-break-artwork)
+- [GIPHY Is Helping Get Artists’ Works Viewed 100 Million Times](https://www.artsy.net/article/artsy-editorial-giphy-artists-works-viewed-100-million-times)
+- [What Makes "Bad" Art Good?](https://www.artsy.net/article/artsy-editorial-bad-art-good)
+
+Articles are responsive, so we look good everywhere:
```

Comment:
Wording is a bit awkward here—I had to re-read it to understand maybe "Articles are responsive, so they look good on all devices:"